packages/backend/src/open_payments/auth/service.ts

```diff
@@ -72,6 +84,53 @@ async function introspectToken(
       }
     )
 
+    const keyType: KeyType = 'public' //dc_TODO
+    const extractable = false //dc_TODO 
+    const typedRequest = requestLike(request)
+    const signatures = httpis.parseSignatures(typedRequest)
```

Comment:
I think this is verifying the backend's token introspection request to the AS, whereas I think we want to validate `ctx.request` that the backend is currently handling.
So this may need to move to the auth middleware, somewhere after the call to `authService.introspect`
https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/auth/middleware.ts#L49